src/Automation/Data/ScanOptions.cs

```diff
@@ -14,6 +14,11 @@ public class ScanOptions
         /// </summary>
         public string ScanId { get; }
 
+        /// <summary>
+        /// The window handle for the root of the UIA subtree to scan.
+        /// </summary>
+        public System.IntPtr WindowHandle { get; set; } = System.IntPtr.Zero;
```

Comment:
`ScanOptions` is intentionally immutable. Rather than adding `ScanTools.SetRootWindowHandle`, maybe pass `ScanTools.ScanRootWindowHandle` as a parameter to `ITargetElementLocator.LocateRootElements` ?

Reply:
Sounds like a reasonable flow to me.  Updating.

Reply:
Added in latest iteration